package.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I don't think this should be in the root directory of the project as it is only related to one of the examples. Could you move it to the particular example's folder? I'm also not sure whether it should be automatically installed with the project.